### PR TITLE
fix code style

### PR DESCRIPTION
**Pull request by razbakov**
I added configuration for automatic code checks: eslint and prettier. 
Please install prettier extension in VSCode and configure auto-format on save.
Your builds will fail if code lint fails.

### CODE REVIEW

**Review by amanuela97 — APPROVED** [review]

File: .prettierrc [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {
   "semi": false,
   "arrowParens": "always",
-  "singleQuote": true
+  "singleQuote": true,
+  "useTabs": false,
+  "tabWidth": 2,
+  "endOfLine": "lf",
```
Comment (VOlekh) [inline .prettierrc:7]: I have end of line CRLF. If I change this in VS Code this will only work till I open some other source file with CRLF.

Comment (VOlekh) [inline .prettierrc:7]: I have end of line CRLF. If I change this in VS Code this will only work till I open some other source file with CRLF.

Comment (VOlekh) [reply .prettierrc:7]: Suggest  'endOfLine': 'auto',

Comment (VOlekh) [reply .prettierrc:7]: Suggest  'endOfLine': 'auto',

Comment (VOlekh) [reply .prettierrc:7]: yarn run lint --fix

**Review by VOlekh — APPROVED** [review]

**Review by harshbrahmxatriya — APPROVED** [review]

File: .prettierrc (hunk 1 above)
Comment (razbakov) [reply .prettierrc:7]: @amanuela97 @harshbrahmxatriya please advice. You had this issue.

**Pull request merged**